vignettes/FAQ.Rmd

```diff
@@ -31,6 +31,28 @@ knitr::opts_chunk$set(
 This document provides answers to common questions about the `rbmi` package.
 It is intended to be read after the `rbmi: Quickstart` vignette.
 
+<br>
+
+
+## Is `rbmi` validated ? 
+
+With regards to software, validation is the act of ensuring that the software meets the needs and
```

Comment:
Maybe to refine the scope slightly
```suggestion
With regards to software in the pharmaceutical industry, validation is the act of ensuring that the software meets the needs and
```

Reply:
The rest of the paragraph is about the pharma industry though, talking about FDA and sponsors, so I think setting the context somewhere would be good.